Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/type/BaseType.java

```diff
@@ -50,7 +50,7 @@
     @SuppressWarnings("unchecked")
     <F extends BaseBean> JSONArray addObjectRelation(List<F> objects) {
         JSONArray jsonArray = new JSONArray();
-        if (objects != null) {
+        if (objects != null && objects.size() > 0) {
```

Comment:
Why did you add this empty check? I think it is not necessary as an empty list will do nothing in later for statement.